.config/prettier.mjs

```diff
@@ -0,0 +1,37 @@
+export default {
+  bracketSpacing: true,
+  endOfLine: 'lf',
+  printWidth: 80,
+  tabWidth: 2,
+  useTabs: false,
+  singleQuote: true,
+  overrides: [
+    {
+      files: [
+        '*.cjs',
```

Comment:
Why do we have cjs files here?

Reply:
This is simply here to cover all of our bases when trying to determine which files should be treated as JS files.